src/zip/fs.ts

```diff
@@ -134,7 +147,8 @@ export class ZipFS<TBuffer extends ArrayBufferLike = ArrayBuffer> extends Readon
 	public constructor(
 		public label: string,
 		protected data: ZipDataSource<TBuffer>,
-		public readonly lazy: boolean = false
+		public readonly lazy: boolean = false,
+		public readonly caseFold?: CaseFold
```

Comment:
Perhaps it is better to leverage an options object now that more parameters are added? For example, having `options: ZipOptions` then assigning to the properities on the instance, or having `protected readonly options: ZipOptions` and using that.